common/djangoapps/oauth_exchange/tests/utils.py

```diff
@@ -101,6 +69,12 @@ def test_confidential_client(self):
             "test_client_id is not a public client"
         )
 
+    def test_inactive_user(self):
+        self.user.is_active = False
+        self.user.save()
+        self._setup_provider_response(success=True)
+        self._assert_success(self.data, expected_scopes=[])
```

Comment:
"Inactive" users are allowed because we are misusing the "is_active" flag to represent email verification, correct?